MediaBrowser.sln

```diff
@@ -1,4 +1,5 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Did this file need to change?

Reply:
Not me - Visual Studio. Can't even find a way to edit that file in VS. Will nuke line 1 in github.

Reply:
Looks like it inserted a BOM.